src/pages/Account/Base/index.js

```diff
@@ -0,0 +1,17 @@
+import template from "./template.hbs";
+import {AbstractComponent} from "../../../components/AbstractComponent";
+import {Avatar} from "../../../components/Avatar";
+
+export class BaseAccountPage extends AbstractComponent {
+    get avatar() {
+        return new Avatar({
+            name: "Сергей"
+        });
+    }
+
+    render() {
+        let html = template(this.props);
+
+        return html.replace("<Avatar />", this.avatar.render());
```

Comment:
Как вариант можно чайлд-строку просто передать контекстом
```suggestion
        return template({
            ...this.props,
            avatar: this.avatar.render(),
        });
```
А в шаблоне просто вставку сделать
`{{avatar}}`